content/intro-to-storybook/vue/pt/get-started.md

```diff
@@ -1,59 +1,57 @@
 
 ```bash
-# Run the test runner (Jest) in a terminal:
+# Execute o test runner (Jest) em um terminal:
 yarn test:unit
 
-# Start the component explorer on port 6006:
+# Inicie o component explorer na porta 6006:
 yarn run storybook
 
-# Run the frontend app proper on port 8080:
+# Execute o app frontend na porta 8080:
 yarn serve
 ```
 
 <div class="aside">
-  Nota: Se <code>yarn test:unit</code>, emitir num erro, poderá ser devido ao facto que o yarn não esteja <a href="https://yarnpkg.com/lang/en/docs/install/">instalado</a>, ou então será necessário instalar o pacote <code>watchman</code>, tal como recomendado neste <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">problema</a>
+  Nota: Se <code>yarn test:unit</code> emitir algum erro, você pode não ter o <a href="https://yarnpkg.com/lang/en/docs/install/">yarn instalado</a>, ou então será necessário instalar o pacote <code>watchman</code>, como recomendado nesta <a href="https://github.com/facebook/create-react-app/issues/871#issuecomment-252297884">issue</a>
 </div>
 
-Sendo estes os seguintes: testes automáticos (Jest), desenvolvimento de componentes (Storybook) e a aplicação em si.
+Nossas três modalidades de app: testes automáticos (Jest), desenvolvimento de componentes (Storybook) e a aplicação em si.
 
 ![3 modalidades](/intro-to-storybook/app-three-modalities.png)
 
-Dependendo de qual parte da aplicação que estamos a trabalhar, podemos querer executar um ou mais simultâneamente.
-Visto que neste caso, o foco é a criação de um componente de interface de utilizador simples, iremos cingir-nos somente á execução de Storybook.
+Dependendo de qual parte da aplicação você está trabalhando, você pode querer executar uma ou mais simultaneamente.
+Visto que nosso foco é a criação de um único componente de UI, iremos ater-nos somente à execução do Storybook.
 
-A Taskbox reutiliza elementos de design do tutorial de React e GraphQL
-[Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), como tal não será necessária a criação de CSS neste tutorial.
-
-O CSS compilado encontra-se disponível [aqui](https://github.com/chromaui/learnstorybook-code/blob/master/src/index.css) e deverá ser adicionado ao seguinte ficheiro `src/index.css` e em seguida importado para a aplicação através da edição do elemento `<style>` no ficheiro `src/App.vue`, de forma a que seja idêntico a:
+## Reutilizar CSS
+A Taskbox reutiliza elementos de design do app exemplo deste [tutorial de React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), então não precisamos escrever CSS neste tutorial. Iremos simplesmente compilar o LESS para um único arquivo CSS e incluí-lo em nosso app. Copie e cole [este CSS compilado](https://github.com/chromaui/learnstorybook-code/blob/master/src/index.css) em `src/index.css` e então importe o CSS dentro do nosso app editando a tag `<style>` em `src/App.vue`, ficando assim:
```

Comment:
This is "another picle" (pardon the bad pun), as when i was going over this section of the tutorial the phrasing might mislead you a bit. We're not actually compiling any LESS. The CSS is already compiled and available, that's why i aimed for providing a more accurate description of what is happening without breaking original content and also avoid a "ipsis verbis" translation.

I think that a better wording choice for this could be something like:

"A Taskbox reutiliza elementos de design deste [Tutorial React e GraphQL](https://blog.hichroma.com/graphql-react-tutorial-part-1-6-d0691af25858), logo não precisamos escrever CSS neste tutorial. Copie e cole [o CSS compilado](https://github.com/chromaui/learnstorybook-code/blob/master/src/index.css) no arquivo ou ficheiro,  `src/index.css` e em seguida importe o CSS para a aplicação editando a tag `<style>` no ficheiro ou arquivo para que fique assim:"